src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
```

Comment:
Regarding field sets, I totally understand removing that bit - I'm torn on it a bit because it's nice to have it as an option for some parts of the query builder, but yet, orderBy for a field set doesn't make sense. It's also a pain in the ass to test methods that rely on field sets because I don't want to create a test field set (and commit to git) just for the sake of running a test. I think I'm going to  remove some of the fieldset methods, but I may keep a few of them for now (like SObjectQueryBuilder.add(FieldSet), I think that one can be handy at times, so I may keep for now).

But yeah, since you've merged your changes now (woohoo!), I'm going to make a few more tweaks to SObjectFieldWrapper, finish up some unit tests and then wrap up this PR. I would definitely like to find a less unseemly way for loading fields into the wrapper, so I will try to include that in this PR if I can.